images/images.qrc

```diff
@@ -767,6 +767,7 @@
         <file>themes/default/mIconAlignRight.svg</file>
         <file>themes/default/mIconArrangeSymbolsLeft.svg</file>
         <file>themes/default/mIconArrangeSymbolsRight.svg</file>
+        <file>themes/default/mIconClearNews.svg</file>
```

Comment:
I'd rename that to mIconClearItem.svg , as I'd love to add this icon to recent projects and templates.